src/main/kotlin/uk/gov/justice/digital/hmpps/manageoffencesapi/repository/OffenceScheduleMappingRepository.kt

```diff
@@ -0,0 +1,15 @@
+package uk.gov.justice.digital.hmpps.manageoffencesapi.repository
+
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.stereotype.Repository
+import uk.gov.justice.digital.hmpps.manageoffencesapi.entity.OffenceScheduleMapping
+import java.util.Optional
+
+@Repository
+interface OffenceScheduleMappingRepository : JpaRepository<OffenceScheduleMapping, Long> {
+  fun findByScheduleParagraphSchedulePartScheduleId(scheduleId: Long): List<OffenceScheduleMapping>
+  fun findByOffenceId(offenceId: Long): List<OffenceScheduleMapping>
+  fun findByOffenceIdIn(offenceId: List<Long>): List<OffenceScheduleMapping>
+  fun deleteByScheduleParagraphIdAndOffenceId(scheduleParagraphId: Long, offenceId: Long)
+  fun findOneByScheduleParagraphIdAndOffenceId(scheduleParagraphId: Long, offenceId: Long): Optional<OffenceScheduleMapping>
```

Comment:
Can't you use Kotling synstax instead of `Optional` i.e. `OffenceScheduleMapping?`

Reply:
yes will do so.. but those methods arent used yet so deleted them (WIP and will be reintroduced in next PR's)